lib/fs.js

```diff
@@ -1903,16 +1932,16 @@ function ReadStream(path, options) {
 
   if (this.start !== undefined) {
     if (typeof this.start !== 'number') {
-      throw new TypeError('"start" option must be a Number');
+      throw new errors.TypeError('ERR_INVALID_OPT_VALUE', 'start', this.start);
```

Comment:
IMHO `ERR_INVALID_ARG_TYPE` might be better.

Reply:
I updated it to `ERR_INVALID_ARG_TYPE`